Bigtable/src/Instance.php

```diff
@@ -273,20 +273,40 @@ public function create(array $clusterMetadataList, array $options = [])
             // `$clustersArray` must be keyed by the cluster ID.
             $clustersArray[$clusterId] = $value;
         }
+        $info = ['displayName' => $displayName ];
+        if ($type != null) {
+            $info['type'] = $type;
+        }
+        if ($labels != null) {
+            $info['labels'] = $labels;
+        }
         $operation = $this->connection->createInstance([
             'parent' => $projectName,
             'instanceId' => $this->id,
-            'instance' => [
-                'displayName' => $displayName,
-                'type' => $type,
-                'labels' => $labels
-            ],
+            'instance' => $info,
             'clusters' => $clustersArray
         ] + $options);
 
         return $this->resumeOperation($operation['name'], $operation);
     }
 
+    /**
+     * Delete an instance.
+     *
+     * Example:
+     * ```
+     * $instance->delete();
+     * ```
+     *
+     * @return array
+     */
+    public function delete()
```

Comment:
Don't forget that any method which triggers a network request must accept `$options = []` as its last argument, and that value must be passed to the connection call.

Reply:
Fixed